tdrs-backend/manifest.buildpack.yml

```diff
@@ -1,7 +1,7 @@
 version: 1
 applications:
 - name: tdp-backend
-  memory: 256M
+  memory: 512M
```

Comment:
This ticket is where I discovered we initially didn't have the headroom for more RAM so I upgraded and then reverted both manifests. Once we got rid of our half-quota and had headroom, I just never re-upgraded the other one since I don't believe we use it, just this "manifest.buildpack.yml". I'll revert the other one back to 512M so it should disappear from this review.